docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source

```diff
@@ -0,0 +1,109 @@
+   where:
+
+   1. **SumoDeployment** is your Sumo Logic deployment that has to be entered in lowercase such as au, ca, de, eu, fed, in, jp, us1, or us2. To find your deployment, see Sumo Logic Endpoints by Deployment and Firewall Security.
+   2. **SumoAccountId** is the Organization ID shown on your Account Overview in the Sumo Logic UI. You can access it by going to **Administration** > **Account** > **Account Overview**.
+
+
+### Step 2. Setup Subscriber in AWS Security Data Lake Console
+
+1. Create a subscriber on Security Data Lake Console. To create a subscriber, follow the instructions below:
+  1. Navigate to the Security Data Lake console.
+  2. From the left navigation, click **Subscriber**.
+  3. On the Subscribers page, under the **Add subscribers** tab click **Create custom subscriber**.
+
+    ![setup-subscriber.png](/img/send-data/setup-subscriber.png)
+
+  4. Optional: Provide the **Subscriber name** and **Description**.
+  5. In the **Select Logs and events sources** section, you can select which resources you want to enable. There are two options given to you:
+     * **All Logs and event sources**. All the logs and event sources will be selected if you select this option.
+     * **Specify log and event sources**. Only specific sources will be selected if you go with this option.
+
+     For example, you select **Specific log and event sources**. In the next step, you will be able to view user activity, API usage in AWS services, IP traffic details, and route S3.
+
+  6. In **Log and event sources**, you see three options again:
+     * **CloudTrail**: View user activity and API usage in AWS services.
+     * **VPC flow logs**: View details about IP traffic to and from network interfaces in your VPC.
+     * **Route S3**
+
+     For example, you select Cloudtrail to get all the user activity and API usage in AWS services.
+
+     ![cloudtrail.png](/img/send-data/cloudtrail.png)
+
+  7. Access data.
+     * **S3** or **Lakeformation** is the storage option you can choose for how your subscribers will access the data.
+
+     :::note
+     Subscription URL will be added to the subscriber in the later step and is kept blank deliberately.
+     :::
+
+
+## Step 3. Create AWS Security Data Lake Source using Sumo Logic Environment
+
+When you create an AWS Security Data Lake source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use from the existing ones or create a new Hosted Collector, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector.md).
+
+To create an AWS Security Data Lake source, follow the instructions below:
+1. In the Sumo Logic environment, go to **Manage Data** > **Collection** > **Collection**.
+2. On the **Collectors page, click **Add Source** next to a Hosted Collector.
+3. Select an existing Hosted Collector or the one you have just created.
+
+   ![aws-security-lake.png](img/send-data/aws-security-lake/png)
```

Comment:
Image link formatting is wrong and this would've broken your build. Please make sure to preview your builds locally prior to submitting a PR.